dcicutils/beanstalk_utils.py

```diff
@@ -865,7 +904,11 @@ def create_foursight(dest_env, bs_url, es_url, fs_url=None):
     if not bs_url.endswith("/"):
         bs_url += "/"
 
-    es_url = es_url.rstrip(":80")
+    if ':80' in es_url:
+        es_url = es_url.rstrip(":80")
+    elif ':443' in es_url:
+        es_url = es_url.rstrip(":443")
```

Comment:
You want to be using misc_utils.remove_suffix.  `.rstrip` takes an argument of a set of characters to strip, not a final substring. `.rstrip(':443')` is the same as `.rstrip('43:')` and is not what you want to do.